scripts/dotnet.py

```diff
@@ -568,6 +580,14 @@ def __process_arguments(args: list):
         help='Download DotNet Cli from the Channel specified.'
     )
 
+    install_parser.add_argument(
```

Comment:
You should add this to `def add_arguments(parser: ArgumentParser) -> ArgumentParser:`. Then, you would not need to duplicate it in the two scripts. It will be added automatically after the call to `dotnet.argument`.

Reply:
I will suggest renaming `--version` to `--dotnet-version` as there might be different One .NET tools added to the mix in the future.

Reply:
Done.